plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -11,51 +11,39 @@
 
 class Test_WebP_Uploads_Picture_Element extends TestCase {
 
-	/**
-	 * Attachment ID.
-	 *
-	 * @var int
-	 */
-	public static $image_id;
-
-	/**
-	 * Setup shared fixtures.
-	 */
-	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ): void {
-		self::$image_id = $factory->attachment->create_upload_object( TESTS_PLUGIN_DIR . '/tests/data/images/leaves.jpg' );
-	}
-
 	/**
 	 * Test that images are wrapped in picture element when enabled.
 	 *
 	 * @dataProvider data_provider_it_should_maybe_wrap_images_in_picture_element
 	 *
-	 * @param bool   $jpeg_and_webp          Whether to enable JPEG and WebP output.
-	 * @param bool   $picture_element        Whether to enable picture element output.
-	 * @param bool   $expect_picture_element Whether to expect the image to be wrapped in a picture element.
-	 * @param string $expected_html          The expected HTML output.
+	 * @param bool   $fallback_jpeg   Whether to fallback JPEG output.
+	 * @param bool   $picture_element Whether to enable picture element output.
+	 * @param string $expected_html   The expected HTML output.
 	 */
-	public function test_maybe_wrap_images_in_picture_element( bool $jpeg_and_webp, bool $picture_element, bool $expect_picture_element, string $expected_html ): void {
+	public function test_maybe_wrap_images_in_picture_element( bool $fallback_jpeg, bool $picture_element, string $expected_html ): void {
 		$mime_type = 'image/webp';
 		if ( ! wp_image_editor_supports( array( 'mime_type' => $mime_type ) ) ) {
 			$this->markTestSkipped( "Mime type $mime_type is not supported." );
 		}
 
-		if ( $jpeg_and_webp ) {
-			$this->opt_in_to_jpeg_and_webp();
+		if ( $fallback_jpeg ) {
+			update_option( 'perflab_generate_webp_and_jpeg', '1' );
 		}
 
 		if ( $picture_element ) {
-			$this->opt_in_to_picture_element();
+			update_option( 'webp_uploads_use_picture_element', '1' );
```

Comment:
Shouldn't this function call be inside of `$this->opt_in_to_picture_element()`?

Reply:
Thanks. Updated tests in https://github.com/WordPress/performance/pull/1408/commits/8e36b2c1d4c47d92c7f151743505fdd10310c336